src/app/app-routing.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import {LandingComponent} from "./landing/landing.component";
```

Comment:
I'd really recommend setting up Prettier (https://prettier.io/) in the repo. Having a unified file structure (same type of quotes used in the project, spacings kept consistent, etc.) really helps out when looking at code.
 `npm install prettier --dev`, set it up and then you can either use "Format File" in all files or set up "Format on Save"

Reply:
I believe I have it